stdlib/public/SwiftShims/swift/shims/_SwiftConcurrency.h

```diff
@@ -16,6 +16,8 @@
 #ifndef SWIFT_CONCURRENCY_H
 #define SWIFT_CONCURRENCY_H
 
+#include <stdlib.h>
```

Comment:
@DougGregor can you please review this change?

Reply:
Actually, I think I am not going to make this change in this PR. I'm just going to ban the one from `stdlib.h`. Otherwise, we will have to update `SILGenModule::getExit` to handle a number of modulemaps/platforms.